buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -726,11 +729,15 @@ class BeamModulePlugin implements Plugin<Project> {
         options.encoding = "UTF-8"
         // As we want to add '-Xlint:-deprecation' we intentionally remove '-Xlint:deprecation' from compilerArgs here,
         // as intellij is adding this, see https://youtrack.jetbrains.com/issue/IDEA-196615
-        options.compilerArgs -= ["-Xlint:deprecation"]
+        options.compilerArgs -= [
+          "-Xlint:deprecation",
+        ]
         options.compilerArgs += ([
           '-parameters',
           '-Xlint:all',
-          '-Werror'
+          '-Werror',
+          '-Xmaxerrs',
+          '10000'
```

Comment:
Is this a typo for maxerrors?

I'm assuming you added this to make it quicker to run, can/should we drop it now?